tests/Feature/GeneratePdfsTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
I think you can remove `IssueArticle`s and only test `DigitalPublicationSection`s. We'll be cleaning out all IssueArticle code from the codebase as it's all deprecated.

Reply:
Gotcha! I also undid the changes I made to `IssueFactory` and `IssueArticleFactory`.